frontend/src/App.css

```
@@ -582,6 +582,330 @@ nav a:hover {
   color: #6b7280;
 }
 
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.5.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)